pkg/features/kube_features.go

```diff
@@ -66,14 +66,6 @@ const (
 	// Enable nodes to change CPUCFSQuotaPeriod
 	CPUCFSQuotaPeriod featuregate.Feature = "CustomCPUCFSQuotaPeriod"
 
-	// owner: @ConnorDoyle, @fromanirh (only for GA graduation)
-	// alpha: v1.8
-	// beta: v1.10
-	// GA: v1.26
-	//
-	// Alternative container-level CPU affinity policies.
-	CPUManager featuregate.Feature = "CPUManager"
```

Comment:
keep it since it was GA-d in 1.26. We need to keep it for two versions

Reply:
Sorry for reply,  Done.